estate/models/estate_property.py

```diff
@@ -36,3 +37,46 @@ class EstateProperty(models.Model):
         copy=False,
         default="new",
     )
+    property_type = fields.Many2one("estate.property.type")
+    buyer = fields.Many2one("estate.property.type", copy=False)
+    seller = fields.Many2one("res.users", default=lambda self: self.env.user)
+    tags_id = fields.Many2many("estate.property.tag")
+    offer_ids = fields.One2many("estate.property.offer", "property_id")
+    total_area = fields.Integer(compute="_compute_total_area")
+    best_price = fields.Float(compute="_compute_best_price")
+
+    @api.depends("garden_area", "living_area")
+    def _compute_total_area(self):
+        for record in self:
+            record.total_area = record.garden_area + record.living_area
+
+    @api.depends("offer_ids")
+    def _compute_best_price(self):
+        for record in self:
+            if len(record.offer_ids.mapped("price")) == 0:
```

Comment:
If you want to check that this property has no offers, you check if the offer_ids field is empty
```suggestion
            if not record.offer_ids:
```